### PR TITLE
Pop text example

### PR DESCRIPTION
**Pull request by elijahsgh**
This is a small example to show users how to add text that pops in over a narrated video.

This type of pop-in text is popular with YouTube Shorts, etc, and Movis made it easy to add animated text.

### CODE REVIEW

**Comment by rezoo** [comment]
Thank you for sending the PR! I cannot work during the daytime on weekends, but I would like to review in detail afterwards. Please wait a little 🏃‍♀️

File: examples/basic/poptext/poptext.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import movis as mv
+import json
+
+scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
+scene.add_layer(
+    mv.layer.Image("background.png"),
+)
+
+scene.add_layer(
+    mv.layer.Audio("output.ogg"),
```
Comment (rezoo) [inline examples/basic/poptext/poptext.py:10]: In my environment, "output.ogg" was not found. Can you check this?

File: examples/basic/poptext/poptext.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import movis as mv
+import json
```
Comment (rezoo) [inline examples/basic/poptext/poptext.py:2]: Personally, I prefer that the import order in Python starts with the standard library followed by third-party libraries. Can you modify it like this?

```python
import json

import movis as mv
```

File: examples/basic/poptext/poptext.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
```
Comment (rezoo) [inline examples/basic/poptext/poptext.py:4]: To align with the consistency of other samples, can you modify the entire line like this?

```python
def main():
    scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
    ...
    scene.write_video("output.mp4")
```

File: examples/basic/poptext/poptext.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    scene[timing["name"]].transform.rotation.init_value = -10.0
+
+    scene[timing["name"]].scale.enable_motion().extend(
+        keyframes=[0.0, 0.25],
+        values=[0.75, 1.0],
+        easings=["ease_in"],
+    )
+
+scene.write_video("output.mp4")
```
Comment (rezoo) [inline examples/basic/poptext/poptext.py:47]: Ditto. To align with the consistency of other samples, can you insert this line?

```python
if __name__ == '__main__':
    main()
```

**Review by rezoo — COMMENTED** [review]
First, thank you for the PR. I think it's generally fine to merge, but I have listed some minor points for revision below. Additionally, it might be preferable to either write a separate python script or provide a more detailed description in the `README.md` on how to generate `output.ogg` using Text-to-speech 👍

File: examples/basic/poptext/poptext.py (hunk 1 above)
Comment (elijahsgh) [reply examples/basic/poptext/poptext.py]: output.ogg is generated by GCP's Text-to-Speech. I could add the generated output.ogg but it's ~200K bin.
I've added the GCP feature.

Comment (rezoo) [reply examples/basic/poptext/poptext.py]: Thanks!

**Comment by rezoo** [comment]
I confirmed that it worked in my environment. There are still some minor tasks such as flake8 and .gitignore, which I would like to implement. Thank you very much!

**Pull request merged**